bcda/auth/backend_test.go

```diff
@@ -140,28 +137,9 @@ func (s *BackendTestSuite) TestHashUnique() {
 	assert.NotEqual(s.T(), hash1.String(), hash2.String())
 }
 
-func (s *BackendTestSuite) TestHashTime() {
```

Comment:
We should still be testing to make sure we get the log alerts if we exceed either bounds.

Reply:
Yes, now that both bounds create logs, we can add Splunk alerts for both cases.